spec/features/schools/ects/register/happy_path_spec.rb

```diff
@@ -102,10 +102,6 @@
     and_i_should_see_the_ect_i_registered
   end
 
-  def when_i_click_change_name
-    page.get_by_role('link', name: 'change name').first.click
```

Comment:
Do you think we should have a reusable set of `when_i_click_link(text)` style methods?